package.json

```diff
@@ -57,13 +57,15 @@
     "@helium/voter-stake-registry-sdk": "0.9.2",
     "@helium/wallet-link": "4.11.0",
     "@jup-ag/api": "^6.0.6",
+    "@keystonehq/keystone-sdk": "^0.8.0",
     "@ledgerhq/hw-app-solana": "7.0.13",
     "@ledgerhq/react-native-hid": "6.30.0",
     "@ledgerhq/react-native-hw-transport-ble": "6.29.5",
     "@ledgerhq/types-devices": "^6.22.4",
     "@maplibre/maplibre-react-native": "^10.0.0-alpha.2",
     "@metaplex-foundation/mpl-bubblegum": "0.6.0",
     "@metaplex-foundation/mpl-token-metadata": "2.10.0",
+    "@ngraveio/bc-ur": "^1.1.13",
```

Comment:
What does this add?

Reply:
Because Keystone follows the protocol specified in https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md and uses animated QR codes for data transmission, in this process, structured data needs to be first encoded and decoded using the UR protocol. Therefore, it is necessary to introduce the bc-ur library to decode the data.

https://github.com/ngraveio/bc-ur